src/i18n/locales/en.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
Can these json-file be formatted?

Reply:
yes its possible eslint provides built in json formatter. done